src/applications/claims-status/containers/Standard5103NoticePage.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import scrollToTop from '@department-of-veterans-affairs/platform-utilities/scrollToTop';
+
+import NeedHelp from '../components/NeedHelp';
+import ClaimsBreadcrumbs from '../components/ClaimsBreadcrumbs';
+import { getClaim as getClaimAction } from '../actions';
+import {
+  claimAvailable,
+  setDocumentTitle,
+  getClaimType,
+} from '../utils/helpers';
+import { setUpPage } from '../utils/page';
+import withRouter from '../utils/withRouter';
+import Default5103EvidenceNotice from '../components/claim-document-request-pages/Default5103EvidenceNotice';
+import { standard5103Item } from '../constants';
+
+const filesPath = '../files';
+const statusPath = '../status';
+
+class Standard5103NoticePage extends React.Component {
```

Comment:
This should be a functional component instead of a class based one

Reply:
I'll update @jerekshoe ! I thought I had to use a class in order to do setDocumentTitle(), setUpPage() and scrollToTop()?

Reply:
Nah you should be fine with a functional component for those

Reply:
updated!